translations/texts/objects/apex/apexcomfychair/apexcomfychair.object.json

```
@@ -92,7 +92,7 @@
     },
     "Texts": {
       "Eng": "This sure is a comfy chair to sit on.",
-      "Rus": ""
+      "Rus": "Это действительно удобный стул для сидения."
```

```suggestion
      "Rus": "На этом стуле наверняка удобно сидеть."
```
А для чего ещё бывают стулья, если не для сидения?